src/app/Shared/Services/Settings.service.tsx

```diff
@@ -90,12 +90,32 @@ export class SettingsService {
     getFromLocalStorage('DATETIME_FORMAT', defaultDatetimeFormat)
   );
 
+  private readonly _theme$ = new BehaviorSubject<ThemeSetting>(getFromLocalStorage('THEME', ThemeSetting.AUTO));
+
   constructor() {
     this._featureLevel$.subscribe((featureLevel: FeatureLevel) => saveToLocalStorage('FEATURE_LEVEL', featureLevel));
     this._visibleNotificationsCount$.subscribe((count: number) =>
       saveToLocalStorage('VISIBLE_NOTIFICATIONS_COUNT', count)
     );
     this._datetimeFormat$.subscribe((format: DatetimeFormat) => saveToLocalStorage('DATETIME_FORMAT', format));
+    this._theme$.subscribe((theme: ThemeSetting) => saveToLocalStorage('THEME', theme));
+  }
+
+  // taken from https://github.com/garygrossgarten/media-query
+  media(query: string): Observable<boolean> {
```

Comment:
I think this observable should emi the MediaQueryList instead. The consumer can just map to something else as needed so that this can be reusable.